pmm-app/src/shared/components/Form/FieldAdapters/FieldAdapters.tsx

```diff
@@ -65,3 +67,25 @@ export const CheckboxFieldAdapter = ({
     </Field>
   );
 };
+
+export const SelectFieldAdapter = ({
+  input, className, options, label, meta, dataQa, ...props
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <Field label={label}>
+      <div data-qa={dataQa}>
+        <Select
+          {...input}
+          {...props}
+          options={options}
+          className={cx(className, styles.input)}
```

Comment:
I would switch `styles.input` and `className`, like this:

```suggestion
          className={cx(styles.input, className)}
```

this way you can override the default style, should need be

Reply:
👌